packages/ember-metal/lib/meta.js

```diff
@@ -460,54 +459,36 @@ if (MANDATORY_SETTER) {
   };
 }
 
-let setMeta, peekMeta;
 
-// choose the one appropriate for given platform
-if (HAS_NATIVE_WEAKMAP) {
-  let getPrototypeOf = Object.getPrototypeOf;
-  let metaStore = new WeakMap();
+let getPrototypeOf = Object.getPrototypeOf;
+let metaStore = new WeakMap();
 
-  setMeta = function WeakMap_setMeta(obj, meta) {
+let setMeta = function WeakMap_setMeta(obj, meta) {
```

Comment:
Since we no longer need to swap based on environment, lets clean things up a bit and make these:

```js
export function setMeta(obj, meta) {
  // ...snip...
}

export function peekMeta(obj, meta) {
  // ...snip...
}
```

Then we can remove the `export { setMeta, peekMeta }` at the end of the file too...

Reply:
Done.